jedi_vim.py

```diff
@@ -417,9 +416,6 @@ def rename():
                 vim.current.window.cursor = r.start_pos
                 vim_command('normal! cw%s' % replace)
 
-            result = new_buffer(window_path)
-            if not result:
-                return
             vim.current.window.cursor = cursor
```

Comment:
@petobens 
Cool.
There are several places in this area that can fail - so you might have gotten confused.. :)
PR will follow soon.